lib/guest_installation_and_configuration_base.pm

```diff
@@ -1830,6 +1879,14 @@ sub check_guest_installation_result_via_ssh {
     $self->get_guest_ipaddr if (($self->{guest_ipaddr_static} ne 'true') and (!($self->{guest_ipaddr} =~ /^\d+\.\d+\.\d+\.\d+$/im)));
     save_screenshot;
     if ($self->{guest_ipaddr} =~ /^\d+\.\d+\.\d+\.\d+$/im) {
+        if ($self->{guest_network_type} eq 'virtual_network') {
+            # Setup dns in /etc/hosts
+            virt_autotest::virtual_network_utils::setup_vm_simple_dns_with_ip($self->{guest_name}, $self->{guest_ipaddr});
+        }
+        if ($self->{guest_installation_method} eq 'import') {
+            # Setup password-less ssh login
+            virt_autotest::utils::ssh_copy_id($self->{guest_name}, default_ssh_key => '/root/.ssh/id_rsa');
+        }
```

Comment:
Both these two if blocks above can be written in one line actually.

Reply:
That will be very long single line. Let's maybe keep it this way .